Clean up the arguments

rminnich opened the pull request:
The pkg flag was not optional, which is kind of confusing.
Make it a required option.

Do a bit of tidying up of command handling and usage.

Much of the erro text was not needed; remove it. The error
in most cases has all you need.

So now it looks like:
rminnich@rminnich-MacBookPro:~/go/src/harvey-os.org/cmd/tmpfs$ ./tmpfs
Usage: tmpfs [options] <tarfile>
  -addr string
    	Network address (default ":5641")
  -debug int
    	Print debug messages
  -net string
    	Default network type (default "tcp4")
rminnich@rminnich-MacBookPro:~/go/src/harvey-os.org/cmd/tmpfs$ ./tmpfs abc
2020/07/28 20:09:45 open abc: no such file or directory
rminnich@rminnich-MacBookPro:~/go/src/harvey-os.org/cmd/tmpfs$

Signed-off-by: Ronald G Minnich <rminnich@gmail.com>

bldybuild commented:
Commits 70c5839fe0e2149d505c0e28c42e133d4bc01503, cd037bce09ec1aecd40d9c91c23d09f5b60549f4, fb017c60bc60f8df771ac2a9119ec55ea915929c, c2d10243688194346f660591fe4159e30a8d20ec, 3bf16444283094bf03a96fa76f3c5dd4c84867b2, 3d6647d6f8d27df84dfc29a4734fcd8a69a36ce6, 5d4267e4803436e17b87e88b13dc79e528299634, d4e936cfd622de322dc93f69144c68bb4c133e21, 011f6c5fa03bdcd433315c60b94f2327beec22fc, e10ee798c48b8570c0fe498a9a34bb4a5bc7d339, 2083b5d673736c065e98643c36dadde76f7593ab, ae26d57f96f38159bba8fed535879d5eb6188655, a5f19812cf0a8be6df68b9f90a8b80d734b6e862, be012e1e2e5fb24c765157fd2ca7e05675d6a1c2, e9bc0c5d244185ab40c3afe20495d49abaef5feb, 09d36a819eb48d6a2c10b36bf654dcf8867d6641, eca7a1343c6d4ac9417784cf5f37419ea05f34d3, 74f9432beadb7179169e08b6c910dd533c62b7c8, 85146fabcdc9609d4ec9ce666b1015dc83fdfcee, 01a7487534d3f8d83a51f100fc25e6a7b08f8413, 25813f9f91c4090f0cd98cd2e79fdeddde6b273d, f2df0ec7dd0552af9a7ee4ab1bf77c302f08bd57, 73becbf9e242fdde41393c39094f7ec132c09e45, 7500b29993f193e68a9785c3b729a45f4080042c, 9e0e6981fc536c9e51ce24c425141a3d09b39e3a, 932794cbf1458f2b23c964c3a50488ff0d47ca75, 50a583966a69d7be86e6e32f0fd842a176e62db4, d3ff5090ddd22d2c8afe12aa07fa1fb0b61588c3, 9c9ed9aa9dae40b97348979cc4b13f6b26624a7e and 90e860f1a8f1412dad5f0c6ae10b650352c3a6b3 are not signed-off.

Please fix these issues:

>	1. quasilyte has not signed-off 70c5839.
>	2. Hana (Hyang-Ah) Kim has not signed-off cd037bc.
>	3. Ilya Tocar has not signed-off fb017c6.
>	4. Hana Kim has not signed-off c2d1024.
>	5. Hana Kim has not signed-off 3bf1644.
>	6. Josh Bleecher Snyder has not signed-off 3d6647d.
>	7. Heschi Kreinick has not signed-off 5d4267e.
>	8. erifan01 has not signed-off d4e936c.
>	9. Hana Kim has not signed-off 011f6c5.
>	10. Matthew Dempsky has not signed-off e10ee79.
>	11. Matthew Dempsky has not signed-off 2083b5d.
>	12. Keith Randall has not signed-off ae26d57.
>	13. Matthew Dempsky has not signed-off a5f1981.
>	14. Ian Lance Taylor has not signed-off be012e1.
>	15. Russ Cox has not signed-off e9bc0c5.
>	16. Russ Cox has not signed-off 09d36a8.
>	17. Russ Cox has not signed-off eca7a13.
>	18. Martin Möhrmann has not signed-off 74f9432.
>	19. Daniel Martí has not signed-off 85146fa.
>	20. Russ Cox has not signed-off 01a7487.
>	21. Ilya Tocar has not signed-off 25813f9.
>	22. Russ Cox has not signed-off f2df0ec.
>	23. Robert Griesemer has not signed-off 73becbf.
>	24. Matthew Dempsky has not signed-off 7500b29.
>	25. Russ Cox has not signed-off 9e0e698.
>	26. Russ Cox has not signed-off 932794c.
>	27. Russ Cox has not signed-off 50a5839.
>	28. Russ Cox has not signed-off d3ff509.
>	29. Ian Lance Taylor has not signed-off 9c9ed9a.
>	30. Russ Cox has not signed-off 90e860f.


If you'd like more information on how to sign your commits please visit [signed-off-by.me](https://signed-off-by.me)

rminnich committed:
Clean up the arguments

The pkg flag was not optional, which is kind of confusing.
Make it a required argument instead.

Do a bit of tidying up of command handling and usage.

Much of the erro text was not needed; remove it. The error
in most cases has all you need.

So now it looks like:
rminnich@rminnich-MacBookPro:~/go/src/harvey-os.org/cmd/tmpfs$ ./tmpfs
Usage: tmpfs [options] <tarfile>
  -addr string
    	Network address (default ":5641")
  -debug int
    	Print debug messages
  -net string
    	Default network type (default "tcp4")
rminnich@rminnich-MacBookPro:~/go/src/harvey-os.org/cmd/tmpfs$ ./tmpfs abc
2020/07/28 20:09:45 open abc: no such file or directory
rminnich@rminnich-MacBookPro:~/go/src/harvey-os.org/cmd/tmpfs$

Signed-off-by: Ronald G Minnich <rminnich@gmail.com>
